### DISCUSSION TITLE
Creating a max Bitvector in Java

### DISCUSSION THREAD

**Discussion by baierd** [cvc5/cvc5#8002] [Q&A] [answered]
Hello,
i use the Java bindings of CVC5 in a [project](https://github.com/sosy-lab/java-smt) and noticed that there is no method anymore to create a (signed/unsigned) max Bitvector term (or i didn't find it, if thats the case please point me to it :D ). Would it be possible to add this in the Java bindings in the future?

Thank you for your work!
Best,
Daniel

**Comment by mpreiner** [comment, marked as answer]
Hi @baierd,
We removed the special methods to create bit-vector values in an attempt to de-clutter the API. You can use mkBitVector(...) to create these values though.

**Comment by baierd** [comment]
Ok good to know. Thank you!
